### PR TITLE
Use less concurrency when sending to reduce the risk of throttling

### PR DESCRIPTION
**Pull request by Marwes**

### CODE REVIEW

**Review by ramn — APPROVED** [review]

**Comment by ramn** [comment]
@Marwes you can merge this and publish as 0.9.0 (put smth in changelog.md) if you are in a rush

File: .clog.toml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[clog]
+repository = "https://github.com/ramn/metrics_cloudwatch"
+
+changelog = "CHANGELOG.md"
+
+from-latest-tag = true
```
Comment (ramn) [inline .clog.toml:6]: Does this require us to tag releases?

File: release.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -ex
+# Usage: ./release.sh patch/minor/major VERSION
+
+
```
Comment (ramn) [inline release.sh]:
```suggestion
cargo install clog-cli && cargo install cargo-release
```

File: .clog.toml (hunk 1 above)
Comment (Marwes) [reply .clog.toml:6]: Yes, but cargo release does that automatically.

**Pull request merged**